### PR TITLE
Data migration

### PR DESCRIPTION
**Pull request by nagendrakommuru**

### CODE REVIEW

File: src/EPR.Calculator.API.Data/bin/Debug/net8.0/EPR.Calculator.API.Data.deps.json [hunk 1]
```diff
@@ -0,0 +1,1470 @@
+{
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/bin/Debug/net8.0/EPR.Calculator.API.Data.deps.json:1]: The bin directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/bin/Debug/net8.0/EPR.Calculator.API.Data.runtimeconfig.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "runtimeOptions": {
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/bin/Debug/net8.0/EPR.Calculator.API.Data.runtimeconfig.json:2]: The bin directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/.NETCoreApp,Version=v8.0.AssemblyAttributes.cs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// <autogenerated />
+using System;
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/obj/Debug/net8.0/.NETCoreApp,Version=v8.0.AssemblyAttributes.cs:2]: The obj directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfo.cs:2]: The obj directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfoInputs.cache [hunk 5]
```diff
@@ -0,0 +1 @@
+7ab900b7ee62734caa4ef82dbf5346e3f1e8fff9c3ae99a5270edef9b1195374
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfoInputs.cache:1]: The obj directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.GeneratedMSBuildEditorConfig.editorconfig [hunk 6]
```diff
@@ -0,0 +1,13 @@
+is_global = true
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.GeneratedMSBuildEditorConfig.editorconfig:1]: The obj directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.GlobalUsings.g.cs [hunk 7]
```diff
@@ -0,0 +1,8 @@
+// <auto-generated/>
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.GlobalUsings.g.cs:1]: The obj directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.CoreCompileInputs.cache [hunk 8]
```diff
@@ -0,0 +1 @@
+8e52c0f87939158bd8f55e7bd1ff0daa727f5202e54abb64ec070cc578e498a8
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.CoreCompileInputs.cache:1]: The obj directory can be added to .gitignore.

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.FileListAbsolute.txt [hunk 9]
```diff
@@ -0,0 +1,15 @@
+C:\GitRepo\epr-calculator-api\src\EPR.Calculator.API.Data\bin\Debug\net8.0\EPR.Calculator.API.Data.deps.json
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.FileListAbsolute.txt:1]: The obj directory can be added to .gitignore.

**Review by johnbenjamin-defra — CHANGES_REQUESTED** [review]
@udaydefra - can you add obj and bin folders to .gitignore please?

File: src/EPR.Calculator.API.Data/bin/Debug/net8.0/EPR.Calculator.API.Data.runtimeconfig.json (hunk 2 above)
Comment (udaydefra) [reply src/EPR.Calculator.API.Data/bin/Debug/net8.0/EPR.Calculator.API.Data.runtimeconfig.json]: Done

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/.NETCoreApp,Version=v8.0.AssemblyAttributes.cs (hunk 3 above)
Comment (udaydefra) [reply src/EPR.Calculator.API.Data/obj/Debug/net8.0/.NETCoreApp,Version=v8.0.AssemblyAttributes.cs]: Done

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfo.cs (hunk 4 above)
Comment (udaydefra) [reply src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfo.cs]: Done

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfoInputs.cache (hunk 5 above)
Comment (udaydefra) [reply src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfoInputs.cache]: Done

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.GeneratedMSBuildEditorConfig.editorconfig (hunk 6 above)
Comment (udaydefra) [reply src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.GeneratedMSBuildEditorConfig.editorconfig]: Done

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.CoreCompileInputs.cache (hunk 8 above)
Comment (udaydefra) [reply src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.CoreCompileInputs.cache]: Done

File: src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.FileListAbsolute.txt (hunk 9 above)
Comment (udaydefra) [reply src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.FileListAbsolute.txt]: Done

**Review by johnbenjamin-defra — APPROVED** [review]

**Pull request merged**